### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_UsingVersionControlinVSCode_Zoom: Content is not visible at 400% zoom and two scroll bars present.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/vscode-docs#3612]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Pre-Requisite:** Change the display resolution to 1280*1024 and zoom in the page to 400% in systems display settings.

**GitHubTags:**#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.4.10;#Win10;#Visual Studio Code;#CELAP1_code;#Zoom;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/editor/versioncontrol
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit)
Chrome Version:Version 81.0.4044.113 (Official Build) (64-bit)
Screen Reader: Narrator(Desktop App), NVDA(2019.3.1)

**Repro Steps:**
1)Launch the application url :
https://code.visualstudio.com/docs/editor/versioncontrol
2)Using Version Control in VS Code Page will get open.
3)Navigate till the heading "Using Version Control in VS Code" while narrator is on.

**Actual:**
issue 1: After applying the pre-requisite steps there are 2 scroll bars present on the screen horizontal and vertical scroll bars.

issue 2: After applying the pre-requisite steps the content present on the screen is overlapped by the cookies pop-up that is present on the screen.

**This issue has been observed throughout the application.**

**Expected:**
issue 1: After applying the pre-requisite steps there should be only 1 scroll bar present on the screen either horizontal or vertical scroll bar.

issue 2: After applying the pre-requisite steps the content present on the screen should be clearly visible and accessible for the users.

**User Impact:**
issue 1: Low vision users won't be able to read data in a single layout.

issue 2: Low vision users might not ne able to access or view the content that is present on the page.

**Attachment link for Reference:**
[3612_CELA P1_A11y_code.visualstudio.com_VisualStudioCode_UsingVersionControlinVSCode_Zoom.zip](https://github.com/microsoft/vscode-docs/files/4531442/3612_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_UsingVersionControlinVSCode_Zoom.zip)

**Comment by gregvanl** [comment]
Closing as duplicate of https://github.com/microsoft/vscode-docs/issues/3432. Behavior is based on browser not code.visualstudio.com

**Comment by ManishaC1** [comment]
Verified the issue in below enivionemnt.
URL:
The issue is fixed.
Github tags:#Closed
![Screenshot (431)](https://user-images.githubusercontent.com/105198219/216943088-0831e5ef-18bc-4abf-b7c3-4799abaf4be7.png)
